index.js

```diff
@@ -429,6 +429,8 @@ FSWatcher.prototype.unwatch = function(paths) {
   paths.forEach(function(path) {
     if (this._closers[path]) {
       this._closers[path]();
+      delete this._closers[path];
+      delete this._getWatchedDir(sysPath.dirname(path)).remove(sysPath.basename(path));
```

Comment:
Consider this `delete` an **oops**. I've missed when I merged some local branches and had a conflict. Sorry about that.